src/shared/defaultAwsContextCommands.ts

```diff
@@ -66,6 +67,7 @@ export class DefaultAWSContextCommands {
         }
 
         await this.loginManager.login({ passive: false, providerId: fromString(profileName) })
+        await vscode.commands.executeCommand('setContext', 'loggedIn', true)
```

Comment:
- is this for future use? I don't see any usages of it in the codebase currently.
- should it be `aws.loggedIn` ?


p.s. AFAIK `setContext` doesn't work in Cloud9 yet.

Reply:
Yes, this was not intended to be included with this change. But that is good to know for future reference as well.